chapters/classes.tex

```diff
@@ -202,7 +202,7 @@ \subsection{Component Declaration Static Semantics}\label{component-declaration-
 
 \subsubsection{Declaration Equations}\label{declaration-equations}
 
-An environment that defines the value of a component of built-in type is said to define a \firstuse{declaration equation}\index{declaration equation}\index{declaration equation!zzzzz@\igobble |seealso{binding equation}} associated with the declared component.
+An environment that defines the value of a component of built-in type is said to define a \firstuse{declaration equation}\index{declaration equation!zzzzz@\igobble |seealso{binding equation}} associated with the declared component.
```

Comment:
```suggestion
An environment that defines the value of a component of built-in type is said to define a \firstuse{declaration equation} associated with the declared component. These are a subset of the binding equations, see \cref{equation-categories}.
```
(I hope I got the section name right.

Reply:
For background see previous thread: https://github.com/modelica/ModelicaSpecification/pull/3043#discussion_r757119274_

Reply:
Having lots of LaTeX hacks in the source is of course not good per se, but on the other hand I think that having _see also_ entries in the index is a good way of making the index more useful.  In this regard, having examples in the source of how it can be done has value in itself, and for this reason I'd like to keep this one as it is.

Reply:
One thing that looks odd with the way it is done currently is the comma at the beginning of the line "see also" line:
![image](https://user-images.githubusercontent.com/20460714/143880629-97e73728-4225-416f-b319-8fa6ac06c4d4.png)
This is in HTML. PDF looks good. I thought it was meant to be after "2nd item" on the line above but somehow it escaped to the next line, but in PDF there is no comma on the line above and no in front of "see also".

Reply:
To me having the "see" in the actual text makes sense here; and I think we should first consider that.

I can understand that we might want to describe this hack somewhere, but keeping it for that reason this doesn't seem like a good solution. Can't we put a link to this hack in some common area (or possibly as a comment in preamble) instead?

If someone were to add "seealso" I wouldn't expect them to scan the entire document to find uses.

Reply:
OK.  I'll do it.

Reply:
Done.  OK now?